DashAI/back/models/classes/sklearn_like_model.py

```diff
@@ -49,13 +49,11 @@ def load_data(self, dataset_path: str, class_column: str) -> dict:
         }
         return prepared_dataset
 
-    def dashai_fit(self, x, y):  # TODO: rename to fit
-        return self.fit(x, y)
+    def fit(self, x, y):
+        return super.fit(x, y)
```

Comment:
I am quite sure that this would not work.
In order to access super's methods, super must be called first.
```suggestion
        return super().fit(x, y)
```

Reply:
Whoops